### DISCUSSION TITLE
Imaging cropping on ios

### DISCUSSION THREAD

**Discussion by akzjcrafts** [saber-notes/saber#1236] [Q&A]
I can’t seem to figure out how to crop images on the iPad app. Is this feature only available through desktop at the moment? Thank you!

**Comment by QubaB** [comment]
This feature is not implemented yet. See #1136 pull request

**Comment by akzjcrafts** [comment]
Oh okay thank you! Sorry I am still trying to figure out how to use GitHub and understand what everything means.
